.github/workflows/mint-release.yml

```diff
@@ -0,0 +1,53 @@
+name: Mint release 🍬
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: Release tag 🏷️
+        required: true
+        default: 0.1.0
+      DOI:
+        description: Reserved DOI from Zenodo
+        required: true
+
+
+jobs:
+  tag:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.12'
+
+    - name: Install dependencies
+      run: |
+        pip install ruamel.yaml
```

Comment:
Not sure if this actually needs `ruamel.yaml`, but it's not an issue to install it anyway.